README.md

```diff
@@ -69,6 +79,19 @@ For ADAL authentication, AZURE_CLIENT_ID and AZURE_CLIENT_SECRET also must be se
 
 5. Run ```mvn clean compile exec:java``` for a sample run through.
 
+## Use latest key vault SDK
+
+The key vault SDK package  here is **com.microsoft.azure.azure-keyvault**, if you are using the [latest](https://search.maven.org/artifact/com.azure/azure-security-keyvault-secrets) version of the key vault SDK package, please reference to the following examples:
+
+* [IdentityReadmeSamples.java](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/keyvault/azure-security-keyvault-secrets/src/samples/java/com/azure/security/keyvault/secrets/IdentityReadmeSamples.java)- Examples to authenticate to key vault secret client
+
+    * createClientCertificateCredential: Create a secret client using cert based authentication.
```

Comment:
There are a lot more ways to authenticate that are shown in this Identity sample.

Reply:
The suggestion above encompasses all this and can be removed.